### PR TITLE
pidgin-sipe: new port

### PR DESCRIPTION
**Pull request by mklein-de**
###### Tested on
macOS 10.12
Xcode 8.3.2

Have you
- [X] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [X] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [X] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [X] tested basic functionality of all binary files?

### CODE REVIEW

File: net/pidgin-sipe/Portfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+version             1.22.0
+categories          net
+platforms           darwin
+license             GPL-2
+maintainers         puffin.lb.shuttle.de:michael.klein openmaintainer
+description         Pidgin protocol plugin for Office 365/Lync/OCS
+long_description    ${description}
+
+homepage            http://sipe.sourceforge.net/
+master_sites        sourceforge:sipe
```
Comment (mojca) [inline net/pidgin-sipe/Portfile]: See https://trac.macports.org/wiki/howto/AvoidRedirects

Comment (mklein-de) [inline net/pidgin-sipe/Portfile]: I updated the Portfile, can this be merged now?

**Comment by mojca** [comment]
Please do the testing with `sudo port -v -t install`. The port is probably missing some dependencies:
```
./configure: line 12328: intltool-update: command not found
checking for intltool >= 0.35.0...  found
configure: error: Your intltool is too old.  You need intltool 0.35.0 or later.
```

File: net/pidgin-sipe/Portfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                pidgin-sipe
+version             1.22.0
+categories          net
+platforms           darwin
+license             GPL-2
+maintainers         puffin.lb.shuttle.de:michael.klein openmaintainer
```
Comment (mojca) [inline net/pidgin-sipe/Portfile]: Please add your github handle:
```
maintainers         {puffin.lb.shuttle.de:michael.klein @mklein-de} openmaintainer
```

**Comment by mojca** [comment]
There were two dependencies missing: `intltool` and `pkgconfig`.

**Comment by mojca** [comment]
The port compiles just on 10.7 and 10.8. On 10.6 and lower there's a broken dependency and on 10.9 and newer it's
```
error: 'gss_display_status' is deprecated: use GSS.framework
```

**Pull request merged**